### PR TITLE
Get rid of $wgWikiaBotUsers

### PR DESCRIPTION
**Pull request by macbre**
@michalroszka / @wladekb

### CODE REVIEW

File: lib/Wikia/tests/Service/User/UserPreferencesIntegrationTest.php [hunk 1]
```diff
@@ -46,8 +47,7 @@ private function getUserPreferences($ignoreCache = false) {
 	}
 
 	function setUp() {
-		global $wgWikiaBotUsers;
-		$this->testUserName = $wgWikiaBotUsers["bot"];
+		$this->testUserName = self::TEST_USER_NAME;
```
Comment (macbre) [inline lib/Wikia/tests/Service/User/UserPreferencesIntegrationTest.php:50]: @harnash

File: includes/wikia/Wikia.php [hunk 2]
```diff
@@ -1341,20 +1341,22 @@ static public function isUnsubscribed( $to, $body, $subject ) {
...
+	 *
+	 * @param User $editor
+	 * @param Title $title
+	 * @return bool return false if you want to block an email
+	 */
+	static public function allowNotifyOnPageChange ( User $editor, /* Title */ $title ) {
+		global $wgWikiaBotLikeUsers;
 
-		return $allow;
+		if ( in_array( $editor->getName(), $wgWikiaBotLikeUsers)  ) {
```
Comment (ghost) [inline includes/wikia/Wikia.php:1354]: how about `return ! in_array( $editor->getName(), $wgWikiaBotLikeUsers);` ?

Comment (macbre) [inline includes/wikia/Wikia.php:1354]: It's the opposite actually

Comment (ghost) [inline includes/wikia/Wikia.php:1354]: Then with `!` :-)

Comment (macbre) [inline includes/wikia/Wikia.php:1354]: I'd keep the if / else approach as the "direct" `return` can be confusing - as the above proves ;)

**Comment by macbre** [comment]
The above is `SwiftStorageTest` fail

File: includes/wikia/Wikia.php (hunk 2 above)
Comment (ghost) [inline includes/wikia/Wikia.php:1354]: OK, then keep it :+1:

**Pull request merged**